docker/tests/Templates/centos7/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM centos:7
+
+# Install dependencies
+RUN yum install -y \
+        curl \
+        glibc-locale-source \
+        git
+
+# Setup the locale
+ENV LANG en_US.UTF-8
+ENV LC_ALL $LANG
+RUN localedef --charmap=UTF-8 --inputfile=en_US $LANG
+
+RUN curl -L -o powershell-{PSVERSIONSTUBRPM}-1.rhel.7.x86_64.rpm {PACKAGELOCATIONSTUB}/powershell-{PSVERSIONSTUBRPM}-1.rhel.7.x86_64.rpm
+RUN yum install -y powershell-{PSVERSIONSTUBRPM}-1.rhel.7.x86_64.rpm
+RUN git clone --recursive https://github.com/PowerShell/PowerShell.git
+RUN pwsh -c "ipmo /PowerShell/build.psm1;exit (Invoke-Pester {TESTLISTSTUB} -PassThru).FailedCount"
```

Comment:
Please use full cmdlet name Import-Module

Reply:
Updated for all files.